example-projects/advanced/phantom/unit-test.js

```diff
@@ -1,10 +1,20 @@
-if (phantom.args.length != 1) {
-    console.log('Expected a target URL parameter.');
-    phantom.exit(1);
+console.log("Phantom version: " + phantom.version.major + "." + phantom.version.minor + "." +  phantom.version.patch);
+if (phantom.version.major > 1) {
+    var args = require('system').args;
+    if (args.length < 2) {
+        console.log('Expected a target URL parameter.');
+        phantom.exit(1);
+    }
+    var url = require('system').args[1];
```

Comment:
Same as above: Shouldn't it be possible here to just get the url from the`args` variable which was declared at line 3?